tools/monorepo-utils/src/workflow-profiler/lib/math.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Calculate the mean value of an array of numbers.
+ *
+ * @param {Array} numbers Array of numbers
+ * @return {number} Mean value
+ */
+export const calculateMean = ( numbers: Array< number > ) => {
+	if ( numbers.length === 0 ) {
+		return 0;
+	}
+
+	const sum = numbers.reduce( function ( a, b ) {
+		return a + b;
+	}, 0 );
+
+	return sum / numbers.length;
+};
+
+/**
+ * Calculate the median value of an array of numbers.
+ *
+ * @param {Array} numbers Array of numbers
+ * @return {number} Median value
+ */
+export const calculateMedian = ( numbers: Array< number > ) => {
```

Comment:
As an undocumented side effect, this modifies the array -- would be cleaner to work on a copy. 

E.g., `const sortedNumbers = [...numbers].sort()`.